bql/udf/builtin/array.go

```diff
@@ -0,0 +1,40 @@
+package builtin
+
+import (
+	"fmt"
+	"gopkg.in/sensorbee/sensorbee.v0/bql/udf"
+	"gopkg.in/sensorbee/sensorbee.v0/core"
+	"gopkg.in/sensorbee/sensorbee.v0/data"
+)
+
+type arrayLengthFuncTmpl struct {
+	singleParamFunc
+}
+
+func (f *arrayLengthFuncTmpl) Call(ctx *core.Context, args ...data.Value) (val data.Value, err error) {
```

Comment:
How about just define this as a function `arrayLength` and use `udf.UnaryFunction` to convert it to a UDF? Then, we can remove some error checks and a panic handling.